### PR TITLE
Create game

### PR DESCRIPTION
**Pull request by ghost**

### CODE REVIEW

File: app/views/layouts/application.html.erb [hunk 1]
```diff
@@ -1,13 +1,12 @@
 <!DOCTYPE html>
 <html>
 <head>
-	<title>IntergalacticBattleship</title>
-	<%= stylesheet_link_tag    "application", media: "all" %>
-	<%= javascript_include_tag "application" %>
+	<title>Intergalactic Battleship</title>
+	<%= stylesheet_link_tag    "application", media: "all", "data-turbolinks-track" => true %>
+	<%= javascript_include_tag "application", "data-turbolinks-track" => true %>
```
Comment (kenmazaika) [inline app/views/layouts/application.html.erb]: One nitpick, but I wouldn't add back in this turbo links stuff.

File: spec/factories.rb [hunk 2]
```diff
@@ -1,21 +1,15 @@
...
-    sequence :email do |n|
-      "rey@example.com"
-    end
-    password "wherearemyparents"
-    password_confirmation "wherearemyparents"
-  end
   
   factory :game do
+    sith_user_id 1
+    jedi_user_id 2
```
Comment (kenmazaika) [inline spec/factories.rb]: These lines above I don't think are doing what you expect.  I think they should be removed.

**Comment by kenmazaika** [comment]
Made a couple small nitpicks, but overall this looks solid.

**Comment by ghost** [comment]
I have removed the turbolinks and the two lines from factories.rb.

**Comment by kenmazaika** [comment]
Looks great

**Pull request merged**